src/vue/backend/src/views/GuidedSetup.vue

```diff
@@ -18,70 +18,65 @@
                         <idx-list-item>Bullet list item 2</idx-list-item>
                         <idx-list-item>Bullet list item 3</idx-list-item>
                     </idx-list>
-                    <idx-button tag="a" href="#" size="lg">Let's Get Started</idx-button>
+                    <idx-button href="#" size="lg" @click="showDialog = true">Let's Get Started</idx-button>
```

Comment:
There is no `showDialog` variable in this file, so it will have errors when clicked